docs/source/markdown/podman-farm-build.1.md.in

```diff
@@ -17,6 +20,9 @@ via emulation using `podman build --arch --platform`.
 
 If no farm is specified, the build will be sent out to all the nodes that `podman system connection` knows of.
 
+Note: Since the images built are directly pushed to a registry, the user must pass in a full image name using the
+**--tag** flag in the format _registry_**/**_repository_**/**_imageName_[**:**_tag_]`.
+
```

Comment:
Please add `@@option tls-verify` to this man page (this one is not technically a blocker, but it will lead to failures in `xref-helpmsgs-manpages` down the line, so, might as well fix now)

Reply:
fixed